examples/filtered_lifting_line/with_correction/alm_simulation.yaml

```diff
@@ -0,0 +1,176 @@
+Simulations:
```

Comment:
Both of these example runs are segfaulting in OpenFAST when I test them locally.  Not sure why. Something about and index of 1 being greater than the max accepted value of 0.  The regression test worked when I tested it.

Reply:
@psakievich I see that you approved this pull-request. Has this been fixed?

Reply:
fixed late last week